### PR TITLE
9.6 release: create changelogs and releases

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* This PR updates the changelogs of the Jetpack plugin and all its dependencies that need an update.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check for typos

### CODE REVIEW

File: projects/packages/connection-ui/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2021-03-30
+### Added
+- Composer alias for dev-master, to improve dependencies
+
+### Changed
+- Connection UI: remove .github directory from production package
+- Pin dependencies
+- Update dependency versions.
```
Comment (anomiex) [inline projects/packages/connection-ui/CHANGELOG.md:15]: Again, line 15 is basically the same thing as line 17.
```suggestion
```

**Comment by jeherve** [comment]
r223565-wpcom

File: projects/packages/licensing/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
...
 
+## [1.4.0] - 2021-03-30
+### Added
+- Composer alias for dev-master, to improve dependencies
+- Dashboard: add new option to input license key.
+
+### Changed
+- Replace usage of deprecated is_active method
+- Update dependency versions.
+- Update package dependencies.
```
Comment (anomiex) [inline projects/packages/licensing/CHANGELOG.md:16]:
```suggestion
- Update package dependencies.
```

File: projects/packages/password-checker/CHANGELOG.md [hunk 3]
```diff
@@ -5,3 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.1.0 - 2021-03-30
+### Added
+- Initial release.
+
+
```
Comment (anomiex) [inline projects/packages/password-checker/CHANGELOG.md:12]: Hmm. How'd this happen?
```suggestion

```
I'll look into fixing changelogger to not do like this again in the future.

File: projects/packages/sync/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Composer alias for dev-master, to improve dependencies
+- Implement a 60 second back-off for non-200 respones, if no retry-after header is present in the response.
+- Impose a max limit of 2MB on post meta values that are synced.
+- Impose a max limit of 5MB on post_content that can be synced.
+
+### Changed
+- Sync: Use the new Password_Checker package instead of Jetpack_Password_Checker.
+- Update dependency versions.
+- Update package dependencies.
```
Comment (anomiex) [inline projects/packages/sync/CHANGELOG.md:18]:
```suggestion
- Update package dependencies.
```

**Review by anomiex — APPROVED** [review]
Don't forget to make sure no one added any new change files to any of these between when you branched and when you merge (:

**Pull request merged**